products/sle/main.pm

```diff
@@ -545,6 +545,7 @@ sub load_consoletests() {
             }
             elsif (!get_var("ZDUP") and !check_var('VERSION', '12')) {    # zypper and sle12 doesn't do upgrade or installation snapshots
                 loadtest "console/installation_snapshots.pm";
+                loadtest "console/snapper_cleanup.pm";
```

Comment:
I made changes according to your suggestions and after a couple of run I realized that in my test I missed to check important part of the concept. The quota group must always keep 50% of file system space. I need to add the login of comparison between file system size and free space available after cleaning up the snapshots. Some googling helped me to find and read this - http://snapper.io/2016/05/18/space-aware-cleanup.html 
The test needs to be updated...